### PR TITLE
French readme parity English

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/pull/2158#issuecomment-444217219

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,7 +121,7 @@ This is needed if you will be using the multi-user mode, to limit access to Fres
 ## Automatic feed update
 * You can add a Cron job to launch the update script.
 Check the Cron documentation related to your distribution ([Debian/Ubuntu](https://help.ubuntu.com/community/CronHowto), [Red Hat/Fedora](https://fedoraproject.org/wiki/Administration_Guide_Draft/Cron), [Slackware](https://docs.slackware.com/fr:slackbook:process_control?#cron), [Gentoo](https://wiki.gentoo.org/wiki/Cron), [Arch Linux](https://wiki.archlinux.org/index.php/Cron)…).
-It is a good idea to run the cron job as the webserver user.
+It is a good idea to run the cron job as the webserver user (often “www-data”).
```
Comment (Frenzie) [inline README.md:124]: Are those intended to be curly quotes?

Comment (Frenzie) [reply README.md:124]: Never mind, I see you changed the quotes into curly quotes all over.

Comment (Alkarex) [reply README.md:124]: Fine with me with `www-data` if you better like it :-)

Comment (Frenzie) [reply README.md:124]: It's fine either way, just making sure it wasn't by accident. :-)

**Pull request merged**